Gemfile

```diff
@@ -22,6 +22,8 @@ gem 'webpacker', '~> 5.0'
 gem 'turbolinks', '~> 5'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 2.7'
+
+gem 'rqrcode'
```

Comment:
I'd like to see us require the `health_cards` gem (just pointed at `path: '.'`) rather than adding its dependencies to the app's Gemfile. I didn't notice anywhere where we were using `rqrcode` outside of the library, but if we are then you can ignore this comment.

Reply:
Removed json-minify and rqrcode from Gemfile and added health_cards reference